kit/azure/meshplatform/README.md

```diff
@@ -0,0 +1,56 @@
+---
+name: Azure meshPlatform
+summary: |
+  integrates this platform with meshStack as a meshPlatform to enable self-service for our engineering teams.
+compliance:
+  - control: cfmm/tenant-management/multi-cloud-tenant-database-integrated-with-lifecycle-management
+    statement: |
+      A central database of all multi-cloud tenants initiates tenant provisioning and deprovisioning processes. The database acts as an authoritative source of tenants and ensures tenant metadata is always up to date.
+---
+
+# Azure meshPlatform
+
+This kit to integrate Azure as a meshPlatform into meshStack instance. With this module, service principals used by meshStack are created with the required permissions. The output of this module is a set of credentials that need to be configured in meshStack as described in [meshcloud public docs](https://docs.meshcloud.io/docs/meshstack.how-to.integrate-meshplatform.html).
```

Comment:
This kit module integrates Azure into meshStack as a platform using the official [terraform-azure-meshplatform](https://github.com/meshcloud/terraform-azure-meshplatform) module. This module sets up service principals and permissions required for meshStack.